robottelo/ui/rhai.py

```diff
@@ -28,14 +29,18 @@ def register_client_to_rhai(self, activation_key, org):
 
         # Red Hat Access Insights requires RHEL 6/7 repo and it not
         # possible to sync the repo during the tests, Adding repo file.
-        self.vm.configure_rhel_repo(conf.properties['clients.rhel6_repo'])
-
-        try:
+        if rhel_distro == 'rhel67':
+            rhel_repo = conf.properties['clients.rhel6_repo']
```

Comment:
@elyezer the configurations are all the same, I have just refactored the code to pass the rhel version for VMs to be spun as an argument to rhai.register_client_to_rhai. In this way,  I will be performing the tests alternately on different VMs rhel67 and rhel71 and the repos will be configured depending on that. earlier, all the tests were running for rhel67 only.